backend/app/glossary/models.py

```diff
@@ -0,0 +1,41 @@
+from datetime import UTC, datetime
+from typing import TYPE_CHECKING
+
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from app.db import Base
+from app.schema import User
+
+if TYPE_CHECKING:
+    from app.schema import User
+
+
+class GlossaryDocument(Base):
```

Comment:
Is it a good idea to spread database schema parts all the over the source tree? It was convenient to have everything inside schema.py

Reply:
It's even worse to store everything in one place. One day the schema file will become unreadable
Project needs directory hierarchy (architecture)

Reply:
Okay, I have no objections on it, just little note: can you use `schema.py` for DB tables? It was named "schema" for DB and "models" for pydantic models and I wanted to save this naming in all parts of the code.